### PR TITLE
P#55975-improve-sorting-controls-17550

### PR DESCRIPTION
**Pull request by hungnc89**
- add box "Sorting" on admin when setting estate and address
- update view select sorting type on estate (jquery)

### CODE REVIEW

File: plugin/Installer/DatabaseChanges.php [hunk 1]
```diff
@@ -228,7 +233,8 @@ private function getCreateQueryListviews()
 			`geo_order` VARCHAR( 255 ) NOT NULL DEFAULT 'street,zip,city,country,radius',
 			`sortBySetting` ENUM('0','1') NOT NULL DEFAULT '0' COMMENT 'Sortierung nach Benutzerwahl: 0 means preselected, 1 means userDefined',
 			`sortByUserDefinedDefault` VARCHAR(200) NOT NULL COMMENT 'Standardsortierung',
-			`sortByUserDefinedDirection` ENUM('0','1') NOT NULL DEFAULT '0' COMMENT 'Formulierung der Sortierrichtung: 0 means highestFirst/lowestFirt, 1 means descending/ascending',
+			`sortByUserDefinedDirection` ENUM('0','1') NOT NULL DEFAULT '0' COMMENT 'Formulierung der Sortierrichtung: 0 means highestFirst/lowestFirt, 1 means descending/ascending',`page_shortcode` tinytext NOT NULL,
```
[the remark this answers is not in the document]
Comment (jayay) [reply plugin/Installer/DatabaseChanges.php:236]: This also leads to a SQL syntax error. I wonder if `dbDelta()` accepts it with a newline-character. See https://github.com/onOfficeGmbH/oo-wp-plugin/runs/4113056535#step:9:14

**Review by jayay — CHANGES_REQUESTED** [review]
Please see my other comments. The tests need to be fixed.

**Comment by michaelarnold-dev** [comment]
@tung-le-esg Could you please check the tests?

**Comment by vu-vuong-esg** [comment]
> @tung-le-esg Could you please check the tests?

We have closed this PR and created the new one in here https://github.com/onOfficeGmbH/oo-wp-plugin/pull/137

**Pull request closed without merging**